plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/LibraryFile.java

```diff
@@ -78,4 +78,30 @@ public boolean isExport() {
   void setExport(boolean export) {
     this.export = export;
   }
+  
+  @Override
+  public String toString() {
+    return this.mavenCoordinates.toStringCoordinates();
+  }
+
+  @Override
+  public int compareTo(LibraryFile other) {
+    return this.mavenCoordinates.toStringCoordinates()
+        .compareTo(other.mavenCoordinates.toStringCoordinates());
+  }
+
+  @Override
+  public boolean equals(Object other) {
+    if (other == null || !(other instanceof LibraryFile)) {
+      return false;
+    }
+    return this.mavenCoordinates.toStringCoordinates()
+        .equals((((LibraryFile) other).mavenCoordinates.toStringCoordinates()));
```

Comment:
Smallish: maybe `mavenCoordinates.compareTo((LibraryFile) other) == 0`?

Reply:
I've always though equals comes first. Guess it's just that I started learning Java before Comparable was a thing. :-)